docs/to_clarify.md

```diff
@@ -2,18 +2,24 @@
 
 ### Forum
 
-1. What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?
-
 1. As mentioned under "Context" that "hotels... have tried different digital check-in solutions to limited success", are you able to show us exactly what are some of the solutions that have been tried and in which aspects are they lacking? Having these references would help us to come up with a better solution.
 
-1. Are there systems depending on the existing check-in/out system? If so, what are the APIs used to communicate between these systems?
-
 1. Can you run through with us the current check-in/out process? This will help us have a better understanding of the pain points experienced when using the existing solution.
 
 1. Stated under the section "Problem Statement" that the solution must be "safe and secured to protect guests' privacy". What exactly does this requirement mean, for example, are there certain security standards that the solution must adhere to?
 
 1. Stated under the section "Possible Use Cases" that the solution should "enhance back-end operational capabilities and business intelligence" which will be used to "create accurate guest profiles and provide personalized and targeted guest experiences". In what manner is the solution expected to do this?
 
+### Integration 
+
+1. What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?
+
+1. Stated under the section "Possible Use Cases" that the solution should "allow data transference capabilities, from room requests collected on the booking engine, to the check-in process till checkout".
+
+Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
```

Comment:
You need to add 4 spaces at the start of the sentence so that this paragraph will be indented properly. Try it out and see the difference.